locales/id/id.json

```diff
@@ -529,7 +529,7 @@
     "There are no available options for this resource.": "Tidak ada opsi yang tersedia untuk sumber ini.",
     "There was a problem executing the action.": "Terdapat masalah saat menjalankan aksi.",
     "There was a problem submitting the form.": "Terdapat masalah saat mengirim form.",
-    "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Orang-orang tersebut telah diundang ke tim Anda dan sudah dikirim email undangan. Mereka bisa bergabung ke dalam tim dengan menerima email undangan.",
+    "These people have been invited to your team and have been sent an invitation email. They may join the team by accepting the email invitation.": "Orang-orang ini telah diundang ke tim Anda dan telah dikirim email. Mereka mungkin bergabung dengan tim dengan menerima email undangan.",
```

Comment:
Both "tersebut" or "ini" are fine. But, for the "may", in this context, "bisa" or "dapat" (which can also means "could") is more fitting than "mungkin" (which means "probably")